src/redux/pattern/pattern.sagas.js

```diff
@@ -94,8 +91,6 @@ function* handlePatternUpdate({ payload }) {
   try {
     yield put(setPatternLoading(true));
     yield call(updatePattern, id, pattern);
-    const patterns = yield call(getAllPatterns, null);
-    yield put(setPatterns(patterns));
     yield put(setSnackBarSeverity('success'));
```

Comment:
I think you need here to dispatch an action to update the pattern in the store

Reply:
it is not necessary to save new pattern in store because immediately after creating new pattern we are redirected to pattern main page. When we go to that page new request is made and we receive all patterns including updated one